src/entities/EditorDocument/index.ts

```diff
@@ -80,6 +80,23 @@ export class EditorDocument {
     return this.#children[index];
   }
 
+  /**
+   * Returns the serialised properties of the EditorDocument.
+   */
+  public get properties(): Record<string, unknown> {
+    return this.#properties;
+  }
+
+  /**
+   * Updates a property of the EditorDocument.
+   *
+   * @param name - The name of the property to update
+   * @param value - The value to update the property with
+   */
+  public updateProperty(name: PropName, value: unknown): void {
+    this.#properties[name] = value;
```

Comment:
could you write a test case of passing unexisting property name?

Reply:
Done